planning/motion_velocity_smoother/CMakeLists.txt

```diff
@@ -47,11 +47,17 @@ rclcpp_components_register_node(motion_velocity_smoother_node
 
 if(BUILD_TESTING)
   ament_add_ros_isolated_gmock(test_smoother_functions
-    test/test_smoother_functions.cpp
+  test/test_smoother_functions.cpp
   )
   target_link_libraries(test_smoother_functions
-    smoother
+  smoother
   )
+  # ament_add_ros_isolated_gtest(test_${PROJECT_NAME}
```

Comment:
May I ask you why it comment out only motion velocity smoother's test?

Reply:
Because implemented test will fail with `motion_velocity_smoother` module...